<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiaAddNewParameter</class>
 <widget class="QDialog" name="DiaAddNewParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font: normal 20px “微软雅黑” ;}

QWidget#diaTestManager {
	background-color: #eaeaea;
}

.QLabel {
	padding: 10px;
	border-radius: 5px;
}

.QPushButton {
	border: 1px solid;
	background-color:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #e9e9e9,stop:1 #f2f2f3);
	padding: 10px;
	border-radius: 5px;
}

.QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

.QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

.QTableWidget{
	color: #000000;
	alternate-background-color: #e0e0e0;
}

.QTableWidget::item:selected {
	color: #000000; 
	background-color: #8fcde6;

}

.QComboBox {
	background-color:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #e9e9e9,stop:1 #f2f2f3);
  	padding: 10;
 	border:1px solid rgba(228,228,228,1);
	border-radius:5px;
	font:20px;
}

.QLineEdit {
	border-radius:5px;
	padding: 10;
	font:20px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>参数名称：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>协议号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_Number"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>结果后缀：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_Suffix"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Add">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>DiaAddNewParameter</receiver>
   <slot>slot_pushButton_Cancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Add</sender>
   <signal>clicked()</signal>
   <receiver>DiaAddNewParameter</receiver>
   <slot>slot_pushButton_Add_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_pushButton_Cancel_clicked()</slot>
  <slot>slot_pushButton_Add_clicked()</slot>
 </slots>
</ui>
